pkg/routes/middleware.go

```diff
@@ -25,19 +25,27 @@ import (
 	"github.com/zinclabs/zinc/pkg/auth"
 )
 
-func AuthMiddleware(c *gin.Context) {
-	// Get the Basic Authentication credentials
-	user, password, hasAuth := c.Request.BasicAuth()
-	if hasAuth {
-		if _, ok := auth.VerifyCredentials(user, password); ok {
-			c.Next()
+func AuthMiddleware(permission string) func(c *gin.Context) {
+	auth.AddPermission(permission)
+	return func(c *gin.Context) {
+		// Get the Basic Authentication credentials
```

Comment:
Comment is unnecessary

Reply:
[mater](https://github.com/zinclabs/zinc/blob/main/pkg/routes/middleware.go#L29)